signer/Cargo.toml

```diff
@@ -15,7 +15,8 @@ description = "Sign extrinsics to be submitted by Subxt"
 keywords = ["parity", "subxt", "extrinsic", "signer"]
 
 [features]
-default = ["sr25519", "ecdsa", "subxt", "native"]
+default = ["sr25519", "ecdsa", "subxt", "std", "native"]
+std = []
```

Comment:
When `std` is enabled we should also enable std on the other crates I think in order to potentially benefit from any std specific things they add